arch/arm64/Kconfig.platforms

```diff
@@ -208,6 +208,24 @@ config ARCH_LAHAINA
 	  LAHAINA chipset. If you do not wish to build a kernel that
 	  runs on this chipset or if you are unsure, say 'N' here.
 
+config ARCH_SONY_NAGARA
+	bool "Sony Mobile Nagaro platform"
+	depends on ARCH_WAIPIO
+	help
+	  Support for the SONY Mobile Nagaro platform
+	  which is based on QCOM WAIPIO chipset.
+	  If you enable this config,
+	  please use SONY Mobile device tree.
+
+config MACH_SONY_PDX223
+	bool "Sony Mobile PDX-223"
+	depends on ARCH_SONY_NAGARA
+	help
+	  Support for the SONY Mobile PDX-223 board
+	  which is based on SONY Nagaro platform.
```

Comment:
`NagarA`, not `NagarO`, everywhere.

Reply:
Why was this marked as resolved?

Reply:
He force pushed. I could not grep the typo in the commit.

Reply:
The typo was fixed in the `config` name, but not in the documentation which is what this comment was about:

![image](https://user-images.githubusercontent.com/2325264/174785173-477b8857-f5f6-41c2-9771-18ca1a705a51.png)